src/framework/midi/internal/platform/osx/coremidioutport.h

```diff
@@ -49,7 +49,7 @@ class CoreMidiOutPort : public IMidiOutPort
     void initCore();
 
     struct Core;
-    std::unique_ptr<Core> m_core;
+    std::shared_ptr<Core> m_core;
```

Comment:
please fix for other platforms: Windows, Linux

Reply:
As @cbjeukendrup works on a Mac, he can't vouch for those I guess, At least he won't see the corresponding issues in his builds

Reply:
I fixed it for Linux

Reply:
Where? Not in alsamidiinport.cpp and alsamidioutport.cpp as far as I can tell, only in the .h files?

Reply:
Hmm, at least not using `make_shared` there

Reply:
https://github.com/musescore/MuseScore/blob/master/src/framework/midi/internal/platform/lin/alsamidiinport.h#L61
https://github.com/musescore/MuseScore/blob/master/src/framework/midi/internal/platform/lin/alsamidiinport.cpp#L49

Reply:
As I wrote: not using `make_shared` in the cpp files

Reply:
See #8899